crates/forge_loader/src/manifest.rs

```diff
@@ -86,11 +157,28 @@ pub struct ForgeModules<'a> {
     scheduled_triggers: Vec<ScheduledTrigger<'a>>,
     #[serde(rename = "consumer", default, borrow)]
     pub consumers: Vec<Consumer<'a>>,
+    #[serde(rename = "compass:dataProvider", default, borrow)]
+    pub data_provider: Vec<DataProvider<'a>>,
+    #[serde(rename = "jira:customField", default, borrow)]
+    pub custom_field: Vec<CustomField<'a>>,
+    #[serde(rename = "jira:uiModificatons", default, borrow)]
+    pub ui_modifications: Vec<UiModificatons<'a>>,
+    #[serde(rename = "jira:workflowValidator", default, borrow)]
+    pub workflow_validator: Vec<WorkflowValidator<'a>>,
     #[serde(rename = "jira:workflowPostFunction", default, borrow)]
-    workflow_post_functions: Vec<WorkflowPostFunction<'a>>,
+    pub workflow_post_function: Vec<WorkflowPostFunction<'a>>,
+    // deserializing admin pages
+    #[serde(rename = "jira:adminPage", default, borrow)]
+    pub jira_admin: Vec<JiraAdminPage<'a>>,
     #[serde(flatten)]
```

Comment:
<!--
 Copyright 2024 Joshua Wong.
 SPDX-License-Identifier: Apache-2.0
-->

We can remove this property, since it's not being used anymore.